src/target/ext/host_io.rs

```diff
@@ -0,0 +1,228 @@
+//! Provide Host I/O operations for the target.
+use crate::arch::Arch;
+use crate::target::{Target, TargetResult};
+use bitflags::bitflags;
+
+bitflags! {
+    /// Host flags for opening files.
+    /// [Open Flags]: https://sourceware.org/gdb/onlinedocs/gdb/Open-Flags.html
+    pub struct HostOpenFlags: u32 {
```

Comment:
you should run `cargo doc` and double-check that this is rendering how you expect. I don't think it does...